src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,89 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final int GROWTH_COEFFICIENT = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
```

Comment:
we don't need this as a class field

Reply:
Do you mean delete all of them as class fields and use them only inside the methods as local variables?

Reply:
I meant threshold only